templates/base/config/jest.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
Can we move the templates/base/config/jest.config.js to templates/base/config/jest/jest.config.js

Reply:
Done